### DISCUSSION TITLE
Querying entries of deactivated sites in GraphQL

### DISCUSSION THREAD

**Discussion by mihob** [craftcms/cms#10609] [Q&A] [answered]
It looks like you can query entries of a sites even though the site itself is disabled.

For example:

```
query {
  entries(id:1,siteId:2) {
    id
  }
}
```

This query would still return a result even though the site with ID 2 would be disabled. Is this working as intended? 
I would expect that entries belonging to disabled sites would not be available to the public.

**Comment by andris-sevcenko** [comment, marked as answer]
Thanks for reporting this - I created an issue for this:  https://github.com/craftcms/cms/issues/10610

**Reply by brandonkelly** [reply]
Craft 4.0.0-beta.4 is out now with the change added in #10610.
